Constitution.txt

```diff
@@ -1,490 +1,264 @@
-  B Franklin
-  Thomas Mifflin
-  Robt Morris
-  Geo. Clymer
-  Thos. FitzSimons 
-  Jared Ingersoll 
-  James Wilson. 
-  Gouv Morris
-
-
-
-Letter of Transmittal
-skip to Letter of Transmittal to Congress   up to the Constitution
-In Convention. Monday September 17th 1787.
-Present
-The States of
-New Hampshire, Massachusetts, Connecticut, Mr. Hamilton from New York, New Jersey, Pennsylvania, Delaware, Maryland, Virginia, North Carolina, South Carolina and Georgia.
+1: The government of any State may, at its discretion and with the assent of its Citizens, according to the State's constitution, elect to leave the Union, and inform the President of these United States in writing, signed by the State's chief executive. 
 
-Resolved, That the preceeding Constitution be laid before the United States in Congress assembled, and that it is the Opinion of this Convention, that it should afterwards be submitted to a Convention of Delegates, chosen in each State by the People thereof, under the Recommendation of its Legislature, for their Assent and Ratification; and that each Convention assenting to, and ratifying the Same, should give Notice thereof to the United States in Congress assembled. Resolved, That it is the Opinion of this Convention, that as soon as the Conventions of nine States shall have ratified this Constitution, the United States in Congress assembled should fix a Day on which Electors should be appointed by the States which shall have ratified the same, and a Day on which the Electors should assemble to vote for the President, and the Time and Place for commencing Proceedings under this Constitution. 
-That after such Publication the Electors should be appointed, and the Senators and Representatives elected: That the Electors should meet on the Day fixed for the Election of the President, and should transmit their Votes certified, signed, sealed and directed, as the Constitution requires, to the Secretary of the United States in Congress assembled, that the Senators and Representatives should convene at the Time and Place assigned; that the Senators should appoint a President of the Senate, for the sole Purpose of receiving, opening and counting the Votes for President; and, that after he shall be chosen, the Congress, together with the President, should, without Delay, proceed to execute this Constitution.
+2: If the exiting State has land borders with a territory which is not a member of the Union, or has water access to the Oceans of the World through waters not claimed as territory by any other State of the Union, the State shall be considered to be sovereign and foreign from these United States on the date one year after the President of these United States has been informed of the State's intent to leave the Union.
 
- 	
-By the unanimous Order of the Convention
+3: Otherwise, the exiting state shall be considered sovereign and foreign on the date two years after the President of these United States has been informed.
```

Comment:
It is completely possible that a landlocked interior state might decide to leave the Union and become an enclave. In such an event, there should be more leadup time, so that the State and the Union can properly negotiate how that State is to have travel through the Union. The Third Reich justified its invasion of Poland partly on the basis of a need to access Prussia. Let such an excuse never be used again.